microdata_tools/_encrypt.py

```diff
@@ -15,6 +15,8 @@
 
 logger = logging.getLogger()
 
+CHUNK_SIZE_BYTES = 250_000_000  # 250 MB per chunk
+
 
 def _encrypt_dataset(rsa_keys_dir: Path, dataset_dir: Path, output_dir: Path) -> None:
```

Comment:
Can these function names be named without a leading underscore as they are used outside the module? 👍🏻 It was done in the _decrypt module but missed here in the last branch 🙇🏻

Reply:
These functions should still be named without the leading underscore 👍🏻 
The module is _-prefixed as it is private to the package, but this function is not private to the module as other modules in the package call it 👍🏻